tdrs-backend/gunicorn_start.sh

```diff
@@ -1,11 +1,29 @@
 #!/usr/bin/env bash
 # Apply database migrations
 set -e
+
+echo "REDIS_SERVER"
+echo $REDIS_SERVER_LOCAL
+if [[ "$REDIS_SERVER_LOCAL" = "TRUE" ]]; then
+    echo "Run redis server on docker"
+else
+    echo "Run redis server locally"
+    export LD_LIBRARY_PATH=/home/vcap/deps/0/lib/:/home/vcap/deps/1/lib:$LD_LIBRARY_PATH
+    ( cd  /home/vcap/deps/0/bin/; ./redis-server /home/vcap/app/redis.conf &)
+fi
+
+#
 echo "Applying database migrations"
 python manage.py makemigrations
 python manage.py migrate
 python manage.py populate_stts
 python manage.py collectstatic --noinput
+
+celery -A tdpservice.settings worker -c 1 --max-memory-per-child 5000 &
+sleep 5
+# celery -A tdpservice.settings --broker=$REDIS_URI flower &
```

Comment:
Is this a To-Do once we work around the RAM issues?

Reply:
Correct, I removed the flower package start command to limit the overhead memory, however it is included in the Pipfile.